pay-api/src/pay_api/utils/util.py

```diff
@@ -43,3 +43,8 @@ def is_valid_redirect_url(url: str):
         if is_valid:
             break
     return is_valid
+
+
+def convert_to_bool(value: str):
```

Comment:
distutils.util.strtobool does this help? Not sure if its any better

Reply:
I will change it